lib/codegen/Decision.cpp

```diff
@@ -82,6 +82,37 @@ void SwitchNode::codegen(Decision *d, llvm::StringMap<llvm::Value *> substitutio
   }
 }
 
+void EqualsLiteralNode::codegen(Decision *d, llvm::StringMap<llvm::Value *> substitution) {
+  std::string funcName;
+  switch (cat) {
+  case SortCategory::Uncomputed:
+  case SortCategory::Map:
+  case SortCategory::List:
+  case SortCategory::Set:
+  case SortCategory::Bool:
+  case SortCategory::MInt:
```

Comment:
Do we expect to support MInt later? Seems like it could be useful for performance.... also Bool.

Reply:
No, those cases are supported by the SwitchLit node.